test/unit/formatting_stack/processors/test_runner/impl.clj

```diff
@@ -22,8 +23,11 @@
 
 (deftest sut-consumers
   (let [corpus (project-namespaces)]
-    (are [input expected] (=  expected
-                              (sut/sut-consumers corpus input))
+
+    (check! (partial < 100) (-> corpus count))
```

Comment:
`100` seems arbitrary,  what are you asserting?

Reply:
> 100 seems arbitrary, what are you asserting?

If it's < 100, something is wrong and the tests will fail rather opaquely